release-notes/7.0/preview/api-diff/preview1/Microsoft.NETCore.App/7.0-preview1_System.Numerics.md

```diff
@@ -0,0 +1,67 @@
+# System.Numerics
+
+``` diff
+ namespace System.Numerics {
+     public static class BitOperations {
++        public static bool IsPow2(IntPtr value);
++        public static bool IsPow2(UIntPtr value);
```

Comment:
@carlossanlop, do we have an issue tracking that the diff doesn't correctly track/handle `nint` and `nuint`? They are showing as `IntPtr` and `UIntPtr` which means that the tooling isn't aware of the attribute.

Reply:
I would assume the tool needs an update to be able to detect these.

This is the tool I used: https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.AsmDiff

First time using it, so I'm unfamiliar with the code. I opened a bug to track it: https://github.com/dotnet/arcade/issues/8484

Reply:
Moving it to the new API Compat object model in the SDK would make these things easier as it uses Roslyn and Roslyn  knows these kind of things already and we don't have to keep updating our CCI code every time a new keyword or feature is added to the language.